sync/test_sync.py

```diff
@@ -0,0 +1,214 @@
+import unittest
+import tempfile
+import shutil
+import ntpath
+import os
+
+from sync import get_links
+from sync import transform_links
+from sync import is_url
+from sync import is_ref
+from sync import remove_ending_forward_slash
+from sync import get_tags
+from sync import get_file_dirs
+from sync import download_files
+from sync import yaml_files_to_list
+from sync import get_files
+
+
+class TestSync(unittest.TestCase):
+
+    # Utils
+
+    def path_leaf(self, path):
+        head, tail = ntpath.split(path)
+        return tail or ntpath.basename(head)
+
+    def read_and_delete_file(self, name):
+        file = open(name, "r")
+        text = file.read()
+        file.close()
+        os.remove(name)
+        return text
+
+    # Tests
+
+    def test_multiple_get_links(self):
+        """ This will ensure that get links will
+        return a list of multiple md links """
+        expected = ["www.link.com", "./link"]
+        result = get_links("this is a [link](www.link.com) and [link](./link)")
+
+        for index, link in enumerate(result):
+            self.assertEqual(link.get("href"), expected[index])
+
+    def test_is_ref(self):
+        """ Verify if a string is a reference. A reference is
+        defined as  a string where its first character is a hashtag """
+        self.assertEqual(False, is_ref(""))
+        self.assertEqual(True, is_ref("#footer"))
+        self.assertEqual(False, is_ref("www.google.com"))
+
+    def test_remove_ending_forward_slash(self):
+        """ Remove a slash if it is the last character in a string """
+        expected = remove_ending_forward_slash("www.google.com/")
+        self.assertEqual("www.google.com", expected)
```

Comment:
Here and in all tests below there is a bit of naming mismatch.
`expected` should be the value that you define as what you expect the function to return, and `actual` is the value that is returned by the function. 

Example of failure:
```
Traceback (most recent call last):
  File "/System/Volumes/Data/go/src/github.com/tektoncd/website/sync/test_sync.py", line 163, in test_is_url
    self.assertEqual(False, expected)
AssertionError: False != True
```
This error message seems to suggest that `expected` is `True` but it's actually the other way around. 

NIT: usually `assertEqual` takes `(actual, expected)`

So this test for instance should be:
```
        expected = "www.google.com"
        self.assertEqual(remove_ending_forward_slash("www.google.com/"), expected)
```